Topics: add details on where to find the classifier model (#3027)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/topics/index.md

model is expected to evolve and improve over time and be updated periodically; the frequency of this
is still under consideration.

#### Where can I find the current classifier model?

> You can find the path to the model file in the "Classifier" tab of `chrome://topics-internals/` page. The top 10k domains currently used are in the `override_list.pb.gz` file found in the same directory as the model above. The domain to topics associations in the list are utilized by the API in lieu of the output of the model itself.

Reviewer: Maybe 'Topics are manually curated for 10,000 top domains, and these are used to train the classifier. The list of these domains and topics can be found in ...'

> You can find the path to the model file in the "Classifier" tab of `chrome://topics-internals/` page. The top 10k domains currently used are in the `override_list.pb.gz` file found in the same directory as the model above. The domain to topics associations in the list are utilized by the API in lieu of the output of the model itself.

Reviewer 2: ```suggestion
Topics are manually curated for 10,000 top domains, and this curation is used to train the classifier. This list can be found in `override_list.pb.gz`, which is available at`chrome://topics-internals/` under the current model in the "Classifier" tab. The domain-to-topics associations in the list are used by the API in lieu of the output of the model itself.
```

> To run the model directly, refer to the documentation here: https://www.tensorflow.org/lite/guide/inference#running_a_model (Also see: https://www.tensorflow.org/learn)

Reviewer 2: ```suggestion
To run the model directly, refer to [TensorFlow's guide to running a model](https://www.tensorflow.org/lite/guide/inference#running_a_model).
```

To inspect the override_list.pb.gz file:

* Unpack it: `gunzip -c override_list.pb.gz > override_list.pb`
* Use protoc to inspect: `protoc --decode_raw < override_list.pb > output.txt`

> * Also see: Taxonomy of topics with IDs: https://github.com/patcg-individual-drafts/topics/blob/main/taxonomy_v1.md

Reviewer 2: ```suggestion

 A full [taxonomy of topics with IDs is available on GitHub](https://github.com/patcg-individual-drafts/topics/blob/main/taxonomy_v1.md).
```
